pkg/app/topaz.go

```diff
@@ -44,6 +45,16 @@ type Topaz struct {
 	Services       map[string]ServiceTypes
 }
 
+var (
+	healthCheck *health.Server
+)
+
+func (e *Topaz) SetServiceStatus(service string, servingStatus grpc_health_v1.HealthCheckResponse_ServingStatus) {
```

Comment:
If we change SetServiceStatus to be a function instead of a method, so change:

```
func (e *Topaz) SetServiceStatus(service string, servingStatus grpc_health_v1.HealthCheckResponse_ServingStatus)
```

to

```
func SetServiceStatus(service string, servingStatus grpc_health_v1.HealthCheckResponse_ServingStatus)
```

so we do not need to pass the app pointer around, and we can call the method using:

```
app.SetServiceStatus("sync", grpc_health_v1.HealthCheckResponse_NOT_SERVING)
```

Reply:
Thanks! Changed this to a func instead of a method so we aren't passing around the app object